Source/WTF/wtf/MathExtras.h

```diff
@@ -494,9 +494,9 @@ inline typename std::enable_if<std::is_floating_point<T>::value, T>::type nanPro
     return std::isnan(a) || std::isnan(b) ? std::numeric_limits<T>::quiet_NaN() : std::max(a, b);
 }
 
-inline bool isIntegral(float value)
+inline constexpr bool isIntegral(const float value)
```

Comment:
The change looks fine overall but please do not mark the parameter as `const` here. In WebKit, we don't use `const` for parameters passed by value.

Reply:
Fixed